test/EDSC/builder-api-test.cpp

```diff
@@ -746,6 +747,41 @@ TEST_FUNC(empty_map_load_store) {
   f.erase();
 }
 
+// CHECK-LABEL: func @affine_if_op
+// CHECK:       affine.if ([[d0:.*]], [[d1:.*]]){{\[}}[[s0:.*]], [[s1:.*]]{{\]}}
+// CHECK-NEXT:  }
```

Comment:
final nit: This 'Check-NEXT: }' doesn't seem necessary.

Reply:
It verifies absence of else block

Reply:
No it doesn't, CHECKs don't check the entire line. This means that if it did have a " else {" afterwards, the check would still pass. Change to `CHECK-NOT: else`, if you want to make sure there is no else.